daq/host.py

```diff
@@ -5,9 +5,12 @@
 import shutil
 import time
 from datetime import timedelta, datetime
+import grpc
 
 from clib import tcpdump_helper
 from report import ResultType, ReportGenerator
+from proto import usi_pb2 as usi
+from proto import usi_pb2_grpc as usi_service
```

Comment:
it's feeling like much of this could/should be extracted to a separate usi module?

Reply:
I don't think it's really necessary at the moment. It brings more complexity but the logic to build these messages types still won't be contained in that usi wrapper class.